Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -78,6 +78,25 @@ extern ITKIOGDCM_EXPORT std::ostream &
  * The compressors supported include "JPEG2000" (default), and
  * "JPEG". The compression level parameter is not supported.
  *
+ * DICOM tags are represented as strings in the metadata
+ * dictionary. The string format is "XXXX,XXXX", DICOM group number followed
+ * by element number, both are hexadecimals. The separator character
+ * is either a pipe "|" or a comma ",".
```

Comment:
Thanks @zivy Do I understand correctly that a pipe "|" is preferable? GDCMImageIOTest2 says that tags are "always read back with the pipe separator": https://github.com/InsightSoftwareConsortium/ITK/blob/cd1fc6499f0dc5e560ec177db4b6b53e286d6659/Modules/IO/GDCM/test/itkGDCMImageIOTest2.cxx#L172-L174

If the pipe is indeed preferable, can you please add this to the documentation as well? For example by extending the last sentence of the warning, below:

```
                                                   [...] To ensure consistency,
 * it is best to always use lower case letters when explicitly adding
 * tags to the metadata dictionary, and use pipe "|" as separator.
```

Reply:
@N-Dekker, nice catch. Thanks.
Will update the PR and force push.